lib/convection/control/stack.rb

```diff
@@ -248,6 +275,22 @@ def validate
         puts "\nTemplate validated successfully"
       end
 
+      def after_create_task(task)
+        @tasks[:after_create].unshift(task)
```

Comment:
@simonirwin-r7 @abegley-r7 and myself discussed this during our daily scrum. We're going to run after tasks in the order they are registered and document that users should define accordingly them if they have a execution dependent order.